### ISSUE TITLE
Use `autocomplete` for relevant input fields

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/web-stories-wp#11211]
## Feature Description

By adding the [`autocomplete`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) attribute (e.g. `autocomplete="url"`) to input fields where it makes sense, we can make users's lives hopefully a bit easier.

**Comment by sblinde** [comment]
@swissspidy As a heads up, Matthew picked this up to work on and came across issues while attempting to do it. As we dug in further, we realized that there's a known issue w/ React (and several other frameworks) where controlled inputs cannot utilize autocomplete. See here: https://github.com/facebook/react/issues/15739

Based on that, I don't think this is something we can accomplish. Adding `autocomplete="on"` does not work in these spaces.

**Comment by swissspidy** [comment]
Oh wow, that's quite the bummer 😞  I didn't know that.

**Comment by sblinde** [comment]
Same, I also wasn't aware, but it's good to know. I'm subscribed to the issue I linked, so if it changes, we can get back to adding this 👍
